### ISSUE TITLE
Announce technical initiatives

### ISSUE THREAD

**Issue by kaywilliams** [ossf/gb-strategy-committee#13]
In our initial press release, we listed the titles of several initial working groups. For this release, let's formalize the current groups, describe the charter for each, and announce progress, e.g. edX course.

Related issues:
- [Create lifecycle for WGs + maturity levels](https://github.com/ossf/tac/issues/13)

**Comment by mayakacz** [comment]
Also related: https://github.com/ossf/tac/issues/9

**Comment by dlorenc** [comment]
Anything I can do to help push this along? It looks like we're waiting on https://github.com/ossf/project-template/pull/1 to be finished up, then each WG to update their READMEs to match the new template. Is that correct? Or are the paragraphs from each one being collected somewhere else?

**Comment by kaywilliams** [comment]
In discussion with @LindsayLF and Jennifer Cloer we decided not to 'announce' initiatives in the press release, but rather to reference the 'Get Involved' page where all initiatives will be listed.

Closing this issue as the Get Involved page is discussed in a separate issue (https://github.com/orgs/ossf/projects/1)
